compiler/src/iree/compiler/Codegen/Common/GPU/test/reduce_bank_conflicts.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt %s --split-input-file --iree-codegen-gpu-reduce-bank-conflicts  | FileCheck %s
+// RUN: iree-opt %s --split-input-file --pass-pipeline="builtin.module(func.func(iree-codegen-gpu-reduce-bank-conflicts))" | FileCheck %s
```

Comment:
This kind of sucks. I wish the pipeline would just apply to `func.func` even though its through the interface. I guess it can get tricky with nested ops implementing the interface.